hydra-web/src/templates/host.selectable.html

```diff
@@ -19,15 +19,15 @@
     <button id="hostRebalanceButton" class='btn btn-default btn-blue btn-hide-zero'>
         Rebalance
     </button>
-    <button id="hostFailFsOkayButton" class='btn btn-default btn-red btn-hide-zero'>
-        Fail (filesystem okay)
-    </button>
-    <button id="hostFailFsDeadButton" class='btn btn-default btn-red btn-hide-zero'>
-        Fail (filesystem dead)
+    <button id="hostFailFsOkayButton" class='btn btn-default btn-blue btn-hide-zero'>
+        Queue to Fail
     </button>
     <button id="hostFailCancelButton" class='btn btn-default btn-blue btn-hide-zero'>
         Cancel failure
     </button>
+    <button id="hostFailFsDeadButton" class='btn btn-default btn-red btn-hide-zero btn-show-one'>
```

Comment:
I didn't separate btn-show-one button and used the current code as much as I can. If I separate it I think I have to add another module to select all button other than "Fail Now!" button.